accurest-core/src/test/groovy/io/codearte/accurest/dsl/WiremockGroovyDslSpec.groovy

```diff
@@ -695,6 +695,8 @@ class WiremockGroovyDslSpec extends WiremockSpec {
 							parameter 'search': $(client(notMatching(~/^\/[0-9]{2}$/)), server("10"))
 							parameter 'age': $(client(notMatching("^\\w*\$")), server(10))
 							parameter 'name': $(client(matching("Denis.*")), server("Denis"))
+							parameter 'credit': $(client(absent()), server("123"))
```

Comment:
Yes, I have just added line for testing without thinking :D it only makes sense absent for all sides